packages/@react-spectrum/progress/docs/ProgressBar.mdx

```diff
@@ -0,0 +1,125 @@
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/progress';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '../package.json';
+
+```jsx import
+import {Flex} from '@react-spectrum/layout';
+import {ProgressBar} from '@react-spectrum/progress';
+import {View} from '@react-spectrum/view';
+```
+
+# ProgressBar
+
+<p>{docs.exports.ProgressBar.description}</p>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['ProgressBar']}
+  sourceData={[
+    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/bar-loader/'}
+  ]} />
+
+## Example
+
+```tsx example
+<ProgressBar label="Loading…" value={50} />
+```
+
+## Value
+
+Progress Bars are controlled with the `value` prop representing current percentage (as an integer).
```

Comment:
I thought there was a slack chat where we decided the names in the docs to match the names in the code, so `ProgressBar` matches `<ProgressBar />`?

Reply:
im happy to do which ever, but we should prob be consistent since the existing packages that were merged (eg StatusLight) also do not do this. let me know which i should do and i'll convert!